website_portal_medical_patient/views/patients_template.xml

```diff
@@ -68,81 +68,104 @@
-                                        t-att-value="request.params.get('birthdate_date') or _format_date(patient.birthdate_date) or ''" />
-                                </div>
+                        </div>
+                        <div class="form-group">
+                            <div t-attf-class="form-field col-xs-12 #{error.get('gender') and 'has-error' or ''} o_website_form_required_custom">
+                                <label class="control-label" for="gender">
+                                    Gender
+                                </label>
+                                <select name="gender" class="form-control o_website_form_input" required="">
+                                    <option value="">Gender . . .</option>
+                                    <t t-foreach="[('female', 'Female'), ('male', 'Male'), ]"
+                                    t-as="gender">
+                                        <option t-att-value="gender[0]"
+                                                t-att-selected="gender[0] == (request.params.get('gender') or patient.gender)">
+                                            <t t-esc="gender[1]" />
+                                        </option>
+                                    </t>
+                                </select>
                             </div>
-                            <div class="form-group">
-                                <div t-attf-class="form-field #{error.get('gender') and 'has-error' or ''} o_website_form_required_custom">
-                                    <label class="control-label label-optional" for="gender">
-                                        Gender
-                                    </label>
-                                    <select name="gender" class="form-control o_website_form_input" required="">
-                                        <option value="">Gender...</option>
-                                        <t t-foreach="[('female', 'Female'), ('male', 'Male'), ]"
-                                        t-as="gender">
-                                            <option t-att-value="gender[0]"
-                                                    t-att-selected="gender[0] == (request.params.get('gender') or patient.gender)">
-                                                <t t-esc="gender[1]" />
-                                            </option>
-                                        </t>
-                                    </select>
-                                </div>
+                        </div>
+                        <div class="form-group">
+                            <div t-attf-class="form-field col-xs-7 #{error.get('weight') and 'has-error' or ''}">
+                                <label class="control-label label-optional" for="weight">
+                                    Weight
+                                </label>
+                                <input type="text"
```

Comment:
Whenever I save this field ("weight") in the frontend, it disappears from the form before saving and isn't saved to the db.  Not sure why.